README.md

```diff
@@ -31,7 +31,7 @@ const plugins = {
   prisma: makePrismaPlugin(),
 };
 
-export const stl = makeStl<{}, typeof plugins>({
+export const stl = new Stl<{}, typeof plugins>({
   plugins,
 });
```

Comment:
I wonder if there's a way to not make the user pass a generic type here? having to pass `plugins` and then also `typeof plugins` feels a little off…

Reply:
It's possible to do that, but it reduces the flexibility of the framework for advanced use-cases. I have a local branch that implements this, but Andy and I aren't sure if it's the right way forward, your input would be very useful here. The solution is that instead of the user passing in a `UserContext` type, if they wish to add context, they extend an `StlUserContext` interface.